packages/zoe/src/zoeService/feePurse.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+
+/**
+ *
+ * @param {Issuer} feeIssuer
+ * @returns {{
+ *   makeFeePurse: MakeFeePurse
+ *   isFeePurse: (feePurse: Purse) => boolean
+ * }}
+ */
+const setupMakeFeePurse = feeIssuer => {
+  const feePurses = new WeakSet();
+
+  /** @type {MakeFeePurse} */
+  const makeFeePurse = () => {
+    const purse = feeIssuer.makeEmptyPurse();
+    /** @type {FeePurse} */
+    const feePurse = Far('feePurse', {
+      ...purse,
+    });
```

Comment:
What is the purpose of wrapping it? Not much takes a purse as an argument, so I'm not sure what hazard we're protecting by hiding the original identity. The one purpose I see it to change the farName, which is an adequate reason. But I'm wondering if you have another purpose in mind?

Reply:
Just to change the farName